src/components/Comments.jsx

```diff
@@ -13,6 +15,10 @@ const Comments = ({currentArticleId}) => {
         })
     }, [comments])
 
+    const handlePostCommentClick = () => {
```

Comment:
You have an infinite loop of requests in this `useEffect`:

On line 13 you are setting `comments`
On line 16 it is running the effect when `comments` is set

🔁